lib/services/android-plugin-build-service.ts

```diff
@@ -0,0 +1,365 @@
+import * as path from "path";
+import * as shell from "shelljs";
+
+const xml2js = require("xml2js");
+
+export class AndroidPluginBuildService implements IAndroidPluginBuildService {
```

Comment:
Consider to add some unit tests for this service.

Reply:
will do